onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportTypeListFragment.java

```diff
@@ -0,0 +1,162 @@
+        setListAdapter(adapter);
+        getListView().setOnItemClickListener(this);
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        ObaAnalytics.reportFragmentStart(this);
+    }
+
+    private Boolean isEmailDefined() {
+        ObaRegion region = Application.get().getCurrentRegion();
+        return !(region == null || region.getContactEmail() == null);
+    }
+
+    @Override
+    public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
+        ReportTypeItem rti = (ReportTypeItem) getListView().getItemAtPosition(i);
+        if (getString(R.string.rt_infrastructure_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity();
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_stop_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_stop));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_arrival_problem).equals(rti.getTitle())) {
+            // Report bus stop issue
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_trip));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_trip_problem));
+        } else if (getString(R.string.rt_app_feedback).equals(rti.getTitle())) {
+            // Send App feedback
+            String email = getString(R.string.ri_app_feedback_email);
+            UIUtils.sendEmail(getActivity(), email, locationString);
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_app_feedback));
+        } else if (getString(R.string.rt_ideas).equals(rti.getTitle())) {
+            // Direct to ideascale website
+            goToIdeaScale();
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_idea_scale));
+        } else if (getString(R.string.rt_customer_service).equals(rti.getTitle())) {
```

Comment:
Could you please re-order the IF statement so it matches the order of the items in the UI?  Makes it a little easier to read.